core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/AvroUpsertMonitor.scala

```diff
@@ -513,8 +517,8 @@ class AvroUpsertMonitorActor(val pollInterval: FiniteDuration,
         throw new RawlsExceptionWithErrorReport(
           RawlsErrorReport(
             StatusCodes.BadRequest,
-            s"All entities failed to update. There were ${failureReports.size} errors in total$additionalErrorString",
-            failureReportsForCaller
+            s"All entities failed to update. There were ${failureReports.size} errors in total$additionalErrorString" +
+              s" Error messages: ${stringMessageFromFailures(failureReportsForCaller, 100)}"
```

Comment:
we used `stringMessageFromFailures` elsewhere, such as in the partial-success case. But here in the total-failure case, we didn't use it … now we do.